### DISCUSSION TITLE
v2/zh/guide/markdown/container

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#479] [Announcements]
# v2/zh/guide/markdown/container

主题可以为你添加提示、注释、信息、注意、警告和详情自定义容器的支持。

https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/container.html

**Comment by kings1990** [comment]
这边最好有这种可以自定义颜色的选项
::: custom (图标) (颜色) 自定义标题

:::

**Reply by Mister-Hope** [reply]
那你完全可以自己写一个，没有人阻止你注册新的 container。
